test/testimportproject.cpp

```diff
@@ -450,7 +450,7 @@ class TestImportProject : public TestFixture {
         project.fileSettings = {std::move(fs1), std::move(fs2)};
 
         project.ignorePaths({"*foo", "bar*"});
-        ASSERT_EQUALS(2, project.fileSettings.size());
+        ASSERT_EQUALS(1, project.fileSettings.size());
```

Comment:
Why does this match less? That obviously looks like a bug.

Reply:
It's because `*` was changed to match any number of characters, including zero. I don't really have a strong opinion on that, but that's how globs usually behave.